hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -113,13 +121,15 @@ public void write(HoodieRecord record, Option<IndexedRecord> avroRecord) {
     Option recordMetadata = record.getData().getMetadata();
     try {
       if (avroRecord.isPresent()) {
+        writeTimer.startTimer();
```

Comment:
If an exception is thrown, the endTimer is not called. Is is ok to call multiple startTimer() without endTimer()?

Reply:
Fixed.